src/Streamly/Internal/Data/SVar.hs

```diff
@@ -229,6 +229,8 @@ data WorkerInfo = WorkerInfo
     }
 
 
+-- /Since: 0.8.0 ("Streamly.Prelude")/
+--
```

Comment:
We need to move this down in the haddock to make it visible in the docs. Also, we should have 0.8.0 as the primary since notation and use `since 0.5.0 (Streamly)` as a historical note. This needs to be updated for all rate operations and all other operations that have been moved.

Reply:
Should historical note be in the haddock?

Reply:
Yes, it should be something like this:

```
@since 0.8.0

/Since 0.1.0 (in Streamly module)/
```

so that users know that the API has been there in the library but was moved in 0.8.0.